README.md

```diff
@@ -49,10 +56,17 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 4. Start running your own analysis!
 
-    <!-- TODO nf-core: Update the example "typical command" below used to run the pipeline -->
+    ```console
+    nextflow run nf-core/hicar -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> -c profile.config
+    ```
+
+    Run it on cluster.
+
+    First prepare a profile config file named as [profile.config](https://nf-co.re/hicar/usage) and a [samplesheet](https://nf-co.re/hicar/usage).
+    Then run:
 
     ```console
-    nextflow run nf-core/hicar -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> --input samplesheet.csv --genome GRCh37
+    nextflow run nf-core/hicar -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> -c profile.config
```

Comment:
https://github.com/nf-core/hicar/pull/1#discussion_r741471586

I don't think that there are loads of pipeline parameters for this pipeline, right? So maybe it's ok to keep the docs here mentioning just the essential bare-minimum command line flags to keep things as simple as possible.

Reply:
I changed it to `-c profile.config`

Reply:
More what I meant was to drop the config file completely and use command line flags instead, to make it as simple and obvious as possible. Taking the pipeline-specific parameters that you have in the example config file then:

```suggestion
    nextflow run nf-core/hicar -profile <docker/singularity/podman/shifter/charliecloud/conda/institute> \
        --input samples.csv \   # Input data
        --qval_thresh 0.01 \    # Cut-off q-value for MACS2
        --genome GRCh38 \       # Genome Reference
        --mappability /path/mappability/bigWig/file  # Provide mappability to avoid memory intensive calculation
```